src/sentry/stacktraces/processing.py

```diff
@@ -257,17 +268,15 @@ def normalize_stacktraces_for_grouping(data: Any, grouping_config: Any = None) -
     """
 
     stacktrace_frames = []
-    stacktrace_exceptions = []
+    stacktrace_container = []
 
     with sentry_sdk.start_span(op=op, description="find_stacktraces_in_data"):
+        # XXX: Why do we want include the raw stacktrace?
```

Comment:
I have been presuming it's so that we can run `_normalize_in_app` on them as well.